sqlparser/parse_test.go

```diff
@@ -1402,6 +1402,15 @@ var (
 		}, {
 			input:   "update test_table set price = price * 1.10 from table2 as t2 where price <= 99.99 returning 1, 0 as literal, t2.zone_id, specified_client_id, other_column, default_client_id, null",
 			dialect: postgresql.NewPostgreSQLDialect(),
+		}, {
+			input:   "prepare fooplan (int, text, bool, bytea) as insert into foo values ($1, $2, $3, $4)",
+			dialect: postgresql.NewPostgreSQLDialect(),
+		}, {
+			input:   "prepare fooplan (int, some_unknown_type) as select * from table_1 where field = $1",
+			dialect: postgresql.NewPostgreSQLDialect(),
+		}, {
+			input:   "execute fooplan ('test', 11, true)",
```

Comment:
could the statement name be wrapped with the quotes? if yes, we should test it too